src/test/java/com/varchenko/MikeHashMapTest.java

```diff
@@ -0,0 +1,40 @@
+package com.varchenko;
+
+import org.junit.jupiter.api.Test;
+
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class MikeHashMapTest {
+    private final MikeHashMap<String, String> mikeHashMap = new MikeHashMap<>();
```

Comment:
Please, use best practices of writing JUnit tests. Think about the initialization objects in the unit tests.

Reply:
Done